sources/whatsnew.asciidoc

```diff
@@ -32,7 +32,7 @@ fine-grained control over the current time and time zone used for validation (se
 * All built-in constraints are marked as repeatable now
 * Parameter names are retrieved using reflection (see <<constraintdeclarationvalidationprocess-methodlevelconstraints-parameterconstraints-namingparameters>>)
 * `ConstraintValidator#initialize()` is a default method (see <<constraintsdefinitionimplementation-validationimplementation>>)
-* The namespace for Bean Validation XML descriptors has been changed to `http://xmlns.jcp.org/xml/ns/validation/configuration` for `META-INF/validation.xml` and `http://xmlns.jcp.org/xml/ns/validation/mapping` for constraint mapping files (see <<validationapi-bootstrapping-xmlconfiguration>>)
+* The namespace for Bean Validation XML descriptors has been changed to `https://xmlns.jakarta.ee/xml/ns/validation/configuration` for `META-INF/validation.xml` and `https://xmlns.jakarta.ee/xml/ns/validation/mapping` for constraint mapping files (see <<validationapi-bootstrapping-xmlconfiguration>>)
```

Comment:
The "What's new" for 2.0 shouldn't change, but there should be a new one for 3.0.